README.md

```diff
@@ -62,15 +62,14 @@ E.g.
 
 *Note: `BACKUP_CRON` and `PRUNE_CRON` are mutually exclusive.*
 
-* `BACKUP_CRON`- A cron expression for when to run the backup. E.g. `0 30 3 * * *` in order to run every night at 3:30 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
+* `BACKUP_CRON` - A cron expression for when to run the backup. E.g. `0 30 3 * * *` in order to run every night at 3:30 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
 * `PRUNE_CRON` - A cron expression for when to run the prune job. E.g. `0 0 4 * * *` in order to run every night at 4:00 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
 * `RESTIC_REPOSITORY` - location of the restic repository. You can use [any target supported by restic](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html). Default `/mnt/restic`
 * `RESTIC_BACKUP_SOURCES` - source directory to backup. Make sure to mount this into the container as a volume (see the example configs). Default `/data`
 * `RESTIC_PASSWORD` - password for the restic repository. Will also be used to initialize the repository if it is not yet initialized
-* `RESTIC_BACKUP_ARGS` - Optional. Additional arguments given to *restic backup*.
-* `RESTIC_BACKUP_TAGS` - Optional. Tags to set on each snapshot, separated by commas. E.g. `swarm,docker-volumes`
```

Comment:
I think as long as we still support this in the code, we should leave `RESTIC_BACKUP_TAGS` in the documentation as well. Just mark it as deprecated here and point to `RESTIC_BACKUP_ARGS` instead.

Reply:
will do